src/core/public/i18n/i18n_service.tsx

```diff
@@ -50,7 +50,7 @@ export class I18nService {
         defaultMessage: 'Select this row',
         description: 'ARIA and displayed label on a checkbox to select a single table row',
       }),
-      'euiBasicTable.tableDescription': ({ itemCount }: EuiValues) =>
+      '˜': ({ itemCount }: EuiValues) =>
```

Comment:
what does this `~` mean?

Reply:
This was a typo, thanks for catching.  Pushed a fix.